chapters/spec-050-clus-class-exercise/exercise.tex

```diff
@@ -0,0 +1,41 @@
+\chapter{Exercises}
+
+\section{Clustering}
+
+Open the “Liver-cirrhosis” data set (\widget{Datasets} widget). It represents an a sliced liver tissue. It was sliced so thin that some parts of the image have very low intensities; let's call these "background".
+
+\begin{enumerate}
+\item Remove the background from this data set. For background removal you are likely to need some preprocessing. Remove the background in any way you see fit. Also clustering, as seen below, could be useful for background removal. Try to make it work.
```

Comment:
change "background" to "background regions"

just so they don't confuse with spectral background.